src/controllers/auth.controller.js

```diff
@@ -93,6 +93,54 @@ class AuthController {
 		ResponseService.setSuccess(200, 'you have successful logged in', token);
 		return ResponseService.send(res);
 	}
+
+	/**
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} resetting the password
+*/
+	static async resetPassword(req, res) {
+		const decodedToken = await TokenService.verifyToken(req.query.token);
+		await UserService.updateUserByAttribute({ email: decodedToken.email }, {
+			password: BcryptService.hashPassword(req.body.password),
+			confirmPassword: BcryptService.hashPassword(req.body.confirmPassword)
+		});
+		if (req.body.password !== req.body.confirmPassword) {
+			ResponseService.setError(400, 'Password does not match!');
+			return ResponseService.send(res);
+		}
+		ResponseService.setSuccess(200, 'Password changed successfully');
+		return ResponseService.send(res);
+	}
+
+	/**
+	 * @param {object} req
+	 * @param {object} res
+	 * @returns {object} helping the user to reset the password
+	*/
+	static async resetpass(req, res) {
```

Comment:
Your naming convetion take an example you named the async function on line 182 resetPassword and this line of 121 resetpass please use better naming convetion of your functions i the controller

Reply:
done